### PR TITLE
Add localization for calendar

### PR DESCRIPTION
**Pull request by Amirroracle**
-Calendar should have localization working now
-Values of the dates are in french 
-Should check if the values saved are in french (email etc..)
-Known issue: if you refresh the page while on the calendar page -> throws an error due to lingui.(fixed)
-Tests need amending - trouble with mocking lingui (withi18n) in Calendar test (done)

### CODE REVIEW

**Comment by Amirroracle** [comment]
[Translate the calendar to FR [M]](https://trello.com/c/vKjZSUtP/134-translate-the-calendar-to-fr-m)

**Comment by Amirroracle** [comment]
[Translate the calendar to FR [M]](https://trello.com/c/vKjZSUtP/134-translate-the-calendar-to-fr-m)

File: web/src/components/__tests__/Calendar.test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { mount } from 'enzyme'
-import { CalendarAdapter } from '../Calendar'
+import CalendarAdapter from '../../components/Calendar'
```
Comment (pcraig3) [inline web/src/components/__tests__/Calendar.test.js]: You can just keep this as `../Calendar`. 

The test file is in the `components` directory, so the filepath is equivalent.

File: web/src/components/__tests__/Summary.test.js [hunk 2]
```diff
@@ -20,8 +20,8 @@ const defaultProps = {
 
 describe('<SelectedDayList />', () => {
   it('renders correct number of rows', () => {
-    const wrapper = shallow(<SelectedDayList selectedDays={selectedDays} />)
-
+    const wrapper = mount(<SelectedDayList selectedDays={selectedDays} />)
+    console.log('aJDYgajsda', wrapper.text())
```
Comment (pcraig3) [inline web/src/components/__tests__/Summary.test.js]: Probably get rid of this too eh?

File: web/src/components/forms/CalendarConstants.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      ],      
+
+    }
+  }
+  return date
+}
+
+/*export const CalendarConstants = withI18n()(({ i18n, language }) => {
+  console.log(i18n._`Review`)
+})*/
```
Comment (pcraig3) [inline web/src/components/forms/CalendarConstants.js]: We can delete this also.

File: web/src/components/__tests__/Time.test.js [hunk 4]
```diff
@@ -3,24 +3,24 @@ import { shallow } from 'enzyme'
...
-    const time = shallow(<Time date={date} />)
+    const time = shallow(<Time date={date} locale='en'/>)
     expect(time.props().dateTime).toEqual('1870-04-22')
-    expect(time.text()).toEqual('Fri, 22 Apr 1870')
+    expect(time.text()).toEqual('Fri, Apr 22, 1870')
   })
 
   it('renders correctly from a date string', () => {
-    const time = shallow(<Time date={dateString} />)
+    const time = shallow(<Time date={dateString} locale='en'/>)
```
Comment (pcraig3) [inline web/src/components/__tests__/Time.test.js]: Can we pass in `fr` and get the french formatting we expect, or what comes out here?

Comment (Amirroracle) [reply web/src/components/__tests__/Time.test.js]: Good idea to add french tests, adding in ```fr``` should work just fine.

File: web/src/components/__tests__/Summary.test.js (hunk 2 above)
Comment (Amirroracle) [reply web/src/components/__tests__/Summary.test.js]: haha done.

File: web/src/components/forms/CalendarConstants.js (hunk 3 above)
Comment (Amirroracle) [reply web/src/components/forms/CalendarConstants.js]: done

**Pull request merged**